dianna/__init__.py

```diff
@@ -56,14 +56,15 @@ def explain_image(model_or_function, input_data, method, labels=(1,), **kwargs):
     return explainer.explain_image(model_or_function, input_data, labels, **explain_image_kwargs)
 
 
-def explain_text(model_or_function, input_data, method, labels=(1,), **kwargs):
+def explain_text(model_or_function, input_data, tokenizer, method, labels=(1,), **kwargs):
```

Comment:
The problem with a default tokenizer is that we're then assuming a specific type of model, which may confuse the user if that's not the type of model they're using. E.g. if we make a Spacy the default, a chemistry model would fail in unexpected ways if the user doesn't provide a chemistry tokenizer.

If tokenizer has no default, we think it would be nicest to have it as positional everywhere. To avoid issues with the labels kwarg it would be nice to have that one positional as well, do you agree? It's actually a bit confusing that we assume a default label.

Reply:
Positional everywhere is also just fine with me. Alternatively we could use a kwarg with a check and error handling if it's default value, None, is not overwritten. I think your option, positional, would be simpler and clearer.

I think I agree about the default label. It only really makes sense when using a binary model. In other cases it's indeed confusing I think. Do you want to change it now or create an issue for later reference? Both are fine with me.